OpenSSL/test/test_ssl.py

```diff
@@ -2376,6 +2376,7 @@ def test_app_data(self):
         :py:obj:`Connection.get_app_data`.
         """
         conn = Connection(Context(TLSv1_METHOD), None)
+        self.assertIsNone(conn.get_app_data())
```

Comment:
I'm relatively new to python. Is this because of the minimum supported python version? If so where would I find that?